no-k8s/Dockerfile

```diff
@@ -0,0 +1,8 @@
+FROM registry.gitlab.factory.social.gouv.fr/socialgouv/docker/kosko:4.2.0
+
+RUN apk add --update --no-cache git=~2
+
+# renovate: datasource=github-tags lookupName=socialgouv/.k8s versioning=1.0.1
+RUN git clone --depth 1 --branch v1.0.1 https://github.com/SocialGouv/.k8s /.k8s
```

Comment:
Looks like one of our [templates](https://github.com/SocialGouv/kosko-charts/tree/master/templates)
We should move those files to `SocialGouv/kosko-chart`

Reply:
Even this image can be generated in the `SocialGouv/kosko-chart` repo 🤔

Reply:
should we really mix kosko-charts (the lib) releases with this template (user-land) releases ?

Reply:
Isn't it a one of the template of the lib ?

Reply:
It looks like so, true. But this one might become a bit more "dynamic".
Let's keep it the way it is at the moment. We'll probably move this `.k8s` folder within `SocialGouv/kosko-chart` a bit later.